src/components/SpaceProposalVotesListItem.vue

```diff
@@ -0,0 +1,118 @@
+<script lang="ts" setup>
+import { computed, ref } from 'vue';
+import { ExtendedSpace, Proposal, Vote, Profile } from '@/helpers/interfaces';
+import { useIntl } from '@/composables';
+import { shorten, getIpfsUrl } from '@/helpers/utils';
+
+const props = defineProps<{
+  space: ExtendedSpace;
+  proposal: Proposal;
+  vote: Vote;
+  profiles: Record<string, Profile>;
+}>();
+
+defineEmits(['openReceiptModal']);
+
+const authorIpfsHash = ref('');
+const relayerIpfsHash = ref('');
+
+const titles = computed(() =>
+  props.proposal.strategies.map(strategy => strategy.params.symbol || '')
+);
+
+const { formatCompactNumber } = useIntl();
+</script>
+
+<template>
+  <div class="flex items-center border-t px-3 py-[14px]">
+    <BaseUser
+      :key="vote.voter"
+      :profile="profiles[vote.voter]"
+      :address="vote.voter"
+      :space="space"
+      :proposal="proposal"
+      width-class="w-[110px] min-w-[110px] xs:w-[130px] xs:min-w-[130px] text-left"
+    />
+
+    <SpaceProposalVotesListItemChoice :proposal="proposal" :vote="vote" />
+    <div
+      class="flex w-[110px] min-w-[110px] items-center justify-end whitespace-nowrap text-right text-skin-link xs:w-[130px] xs:min-w-[130px]"
+    >
+      <span
+        v-tippy="{
+          content: vote.scores
+            ?.map(
+              (score, index) => `${formatCompactNumber(score)} ${titles[index]}`
+            )
+            .join(' + ')
+        }"
+      >
+        {{
+          `${formatCompactNumber(vote.balance)} ${shorten(
+            proposal.symbol || space.symbol,
+            'symbol'
+          )}`
+        }}
+      </span>
+      <BasePopover>
```

Comment:
We should use centered titles like everywhere else 
<img width="450" alt="image" src="https://user-images.githubusercontent.com/51686767/226078341-c31cc418-06f0-4e76-b164-74e8937c4a39.png">